packages/navbrand/src/react/index.js

```diff
@@ -0,0 +1,85 @@
+import { css, media } from 'glamor'
+import Halo from '@pluralsight/ps-design-system-halo'
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import filterReactProps from '@pluralsight/ps-design-system-filter-react-props'
+
+import stylesheet from '../css/index.js'
+
+const styles = {
+  logo: props =>
+    css(
+      stylesheet['.psds-navbrand__logo'],
+      props.wordmark &&
+        media(
+          '(min-width: 1200px)',
+          stylesheet['@media (min-width: 1200px)'][
+            '.psds-navbrand__logo--wordmark'
+          ]
+        )
+    ),
+  navBrand: props =>
+    css(
+      stylesheet['.psds-navbrand'],
+      (props.href || props.onClick) && stylesheet['.psds-navbrand--clickable']
+    ),
+  wordmark: () =>
+    css(
+      stylesheet['.psds-navbrand__wordmark'],
+      media(
+        '(min-width: 1200px)',
+        stylesheet['@media (min-width: 1200px)']['.psds-navbrand__wordmark']
+      )
+    )
+}
+
+const NavBrand = React.forwardRef((props, forwardedRef) => {
+  const ref = React.useRef()
+  React.useImperativeHandle(forwardedRef, () => ref.current)
+
+  const tagName = props.href ? 'a' : props.onClick ? 'button' : 'div'
+
+  return (
+    <Halo inline gapSize={Halo.gapSizes.small}>
+      {React.createElement(
+        tagName,
+        {
+          ...styles.navBrand(props),
+          ...filterReactProps(props, { tagName }),
+          ref
+        },
+        <>
+          <Logo {...props} />
+          <Wordmark {...props} />
```

Comment:
Even though we don't currently have the heaviest weight of Gotham, I think we should use text here. The weight issue can be addressed and using text will help keep the usage more consistent and accessibile.

```
color: #fff;
font-weight: 900;
font-size: 20px;
letter-spacing: -.03em;
line-height: 24px;
```

Reply:
done